### ISSUE TITLE
How can I add unofficial list of 3rd party boards support urls?

### ISSUE THREAD

**Issue by Heltec-Aaron-Lee** [arduino/Arduino#11524]
I can't download 3rd party board json file anymore.
![image](https://user-images.githubusercontent.com/32860565/118100268-d880f480-b408-11eb-94cf-808969fae4d7.png)

How can I add 3rd party boards json files to this list?
https://github.com/arduino/Arduino/wiki/Unofficial-list-of-3rd-party-boards-support-urls

**Comment by per1234** [comment]
Thanks for your report @Heltec-Aaron-Lee 

Here's the URL:
https://resource.heltec.cn/download/package_CubeCell_index.json

I am able to reproduce the issue with Arduino IDE 1.8.14
I am not able to reproduce the issue with Arduino CLI 0.18.2:
```
arduino-cli core update-index --additional-urls https://resource.heltec.cn/download/package_CubeCell_index.json
```

I have not been able to identify what might be the cause of the problem. If I download the file and then host it on GitHub then it works fine for me.

**Comment by Heltec-Aaron-Lee** [comment]
Yes, the json file in GitHub also fine for us, but the json from other servers will cause an error.

**Comment by per1234** [comment]
When you say "other servers", do you have other examples of the same problem happening with different URLs, or is it limited only to https://resource.heltec.cn/download/package_CubeCell_index.json ?

**Comment by Heltec-Aaron-Lee** [comment]
such as these:

https://resource.heltec.cn/download/package_CubeCell_index.json
https://resource.heltec.cn/download/package_heltec_esp32_index.json

**Comment by cers000** [comment]
Now，2025-2-17 22:32:15，the 3rd party boards support urls is delete, wiki page display create new page
